misc/kafka-util/summarize_topics.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+"""summarize_topics
+
+Basic script to print messages count and number of bytes within a Kafka topic
+"""
+
+import argparse
+import fnmatch
+import time
+
+import kafka
+
+
+def summarize_topic(args, topic):
+    """Read messages from a topic and print basic information about the topic."""
```

Comment:
since this depends on the other PR, this should need to be updated with typing information when you update pycheck.

Reply:
👍🏻 Adding type checking now and updating `pycheck` file too.